lambda_functions/tests/helper/setup_scenarios.py

```diff
@@ -42,32 +57,51 @@ def create_users_with_different_experience_levels(session) -> Session:
 
     users_to_create = []
```

Comment:
Wouldn't it be easier to populate this list directly with user objects instead of dictionaries which are then later transformed into user objects? This way you could simply use `session.add_all` instead of iterating over the list again.

Reply:
In any case! Just wanted to change existing code as little as possible, so I adopted this structure. 
Think the iteration over the list has been made to avoid to create user objects already present in the db.
`if(not user_already_exists) ... session.merge(new_user)`